performance-tests/performance.test.js

```diff
@@ -93,8 +106,8 @@ describe('Tram-One - Performance Tests', () => {
 		// we'll use the median, since that is the least prone to error from outliers
 		// we'll also add a buffer to the right, to account for fragility
 		const buffer = 1 // 1 seconds, which will map to 5 seconds on the left
-		expect(stats['0050'].medianTime / 5).not.toBeGreaterThan(stats['0010'].medianTime + buffer)
-		expect(stats['0500'].medianTime / 5).not.toBeGreaterThan(stats['0100'].medianTime + buffer)
-		expect(stats['5000'].medianTime / 5).not.toBeGreaterThan(stats['1000'].medianTime + buffer)
+		expect(stats['0050'].averageTime / 5).not.toBeGreaterThan(stats['0010'].averageTime + buffer)
+		expect(stats['0500'].averageTime / 5).not.toBeGreaterThan(stats['0100'].averageTime + buffer)
+		expect(stats['5000'].averageTime / 5).not.toBeGreaterThan(stats['1000'].averageTime + buffer)
```

Comment:
using `averageTime` instead of `medianTime`, since that should be the least error prone (with the outliers removed)